src/AppBundle/RequestSql/RequestSql.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace AppBundle\RequestSql;
+
+class RequestSql
+{
+    protected $em;
+    private $rqSql;
+
+    /***
+     * RequestSql constructor.
+     * @param \Doctrine\ORM\EntityManager $em
+     */
+    public function __construct(\Doctrine\ORM\EntityManager $em)
+    {
+        $this->em = $em;
+        $this->rqSql = array();
+    }
+
+    /***
+     * @return array
+     */
+    public function getRqSql()
+    {
+        return $this->rqSql;
+    }
+
+    /***
+     * @param $value
+     */
+    public function setRqSql($value)
+    {
+        $this->rqSql = $value;
+    }
+
+    /***
+     * @param $rqSql
+     * @return array
+     */
+    public function processingSql($rqSql)
+    {
+
+        $dataChart = array();
+        $xAxis = array();
+
+        //If the request SQL type by user begin by "select"
+        $testRequest = explode(" ", $rqSql);
+
+        if(strtoupper($testRequest[0]) == "SELECT")
```

Comment:
Maybe you have to try another way to secure SQL injection.
Check this for example https://www.w3schools.com/sql/sql_injection.asp
💡 http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/native-sql.html